README.markdown

```diff
@@ -1,9 +1,13 @@
 # fortune.py
 
+This is a fork of
```

Comment:
Not if you're making a pull request!

Reply:
I added this to make clear, that the repository in my account is a fork.
I should have left out README.md

Reply:
Yeah, and and I appreciate you linking back to the original project. It was just a problem for the pull request, since that would have made it back in here.